documentation/src/main/wiki/Knot.md

```diff
@@ -1,5 +1,99 @@
+| `clientRequest.path`                 | `String`                      | &#10004;       | client request url |
+| `clientRequest.method`                 | `HttpMethod`                      | &#10004;       | client request method |
+| `clientRequest.headers`                 | `MultiMap`                      | &#10004;       | client request headers |
+| `clientRequest.params`                 | `MultiMap`                      | &#10004;       | client request parameters |
+| `clientRequest.formAttributes`                 | `MultiMap`                      |       | form attributes, relevant to POST requests |
+| `clientResponse.statusCode`                 | `HttpResponseStatus`                      |   &#10004;    | `OK` status |
+| `clientResponse.headers`                 | `MultiMap`                      | &#10004;       | client response headers |
+| `clientResponse.body`                 | `Buffer`                      |        | response body, can be empty at some routing steps (Knots), finally produced based on fragments |
+| `fragments`                 | `List<Fragment>`                      |   &#10004;    | list of Fragments created with Splitter module |
+| `transition`                 | `String`                      |        | empty |
+
+
+#### Knot response model
+Table below represents Knot Context response.
+
+| Name                        | Type                                | Mandatory | Description  |
+|-------:                     |:-------:                            |:-------:  |-------|
+| `clientRequest.path`                 | `String`                      | &#10004;       | client request url |
+| `clientRequest.method`                 | `HttpMethod`                      | &#10004;       | client request HTTP method |
+| `clientRequest.headers`                 | `MultiMap`                      | &#10004;       | client request HTTP headers |
+| `clientRequest.params`                 | `MultiMap`                      | &#10004;       | client request parameters |
+| `clientRequest.formAttributes`                 | `MultiMap`                      |       | form attributes, relevant to POST requests |
+| `clientResponse.statusCode`               | `HttpResponseStatus`                      |    &#10004;    | `OK` to process routing, other to beak routing  |
+| `clientResponse.headers`                 | `MultiMap`                      | &#10004;       | client response HTTP headers, can be updated by Knot |
+| `clientResponse.body`                 | `Buffer`                      |        | response body for client, can be empty for some routing steps (Knots), finally produced based on fragments |
+| `fragments`                 | `List<Fragment>`                      |   &#10004;    | list of Fragments created with Splitter module |
+| `transition`                 | `String`                      |        | defines next routing step (Knot), empty for redirects, errors and last routing steps |
+
+
+##### Next routing step response
```

Comment:
Lets add:
`##### Examples`, then `Next routing step response`, `Redirect response` and `Error response` may be just **bolded**.